build.sbt

```diff
@@ -0,0 +1,7 @@
+name := "weatherapp-scala"
+version := "0.0.1"
+scalacOptions := Seq("-unchecked", "-feature", "-deprecation", "-encoding", "utf8", "-Xfatal-warnings", "-language:postfixOps")
+mainClass := Some("weatherapp.WeatherApp")
```

Comment:
This is not correct. That package location is not valid. If you run an `sbt run` it'd fail.

It should be: `Some("sbousamra.weatherapp.WeatherApp")`